electron/src/lib/openGraph.ts

```diff
@@ -61,7 +61,6 @@ const fetchImageAsBase64 = async (url: string): Promise<string | undefined> => {
   try {
     response = await axiosWithCookie<Buffer>(axiosConfig);
   } catch (error) {
-    logger.error(error);
```

Comment:
Removed this to log this only once (will be logged by the webapp later) and to avoid errors in the console which should be warnings.